app/helpers/breadcrumb_helper.rb

```diff
@@ -79,4 +79,9 @@ def allocations_breadcrumb
     path = provider_recruitment_cycle_allocations_path(@provider.provider_code, @provider.recruitment_cycle_year)
     provider_breadcrumb << ["Request PE courses for 2021/22", path]
   end
+
+  def allocations_closed_breadcrumb
```

Comment:
is this actually used?

Reply:
yes, its used here https://github.com/DFE-Digital/publish-teacher-training/pull/1245/files#diff-35db6979acf365c20f12b24a77baa3b4R2